recipes/osgearth/all/conanfile.py

```diff
@@ -84,6 +86,8 @@ def configure(self):
             del self.options.fPIC
 
     def config_options(self):
+        self.options["openscenegraph"].shared = self.options.shared
```

Comment:
Openscenegraphs plugin system may require some tinkering until everything works, especially when mixing shared and dynamic builds of plugins or openscenegraph. 
While not strictly necessary, I think it would be nice to have the same linkage as osgearth by default (right now shared osgearth version links to static openscenegraph).

Reply:
> While not strictly necessary,

Then do not customize it. You still can do it locally. We only enforce an option when the default configuration is completely broken and does not work. It also avoids the expected behavior for users. Please, revert it.